.circleci/config.yml

```diff
@@ -24,11 +24,10 @@ install_platform_cli: &install_platform_cli
     sudo ln -s ~/.platformsh/bin/platform /usr/local/bin/platform
     # Add platform.sh remote to git.
     if ! git config remote.platform.url > /dev/null; then
-      git remote add platform $PLATFORMSH_PROJECT_NAME@git.eu.platform.sh:$PLATFORMSH_PROJECT_NAME.git
+      platform project:set-remote $PLATFORMSH_PROJECT_NAME
     fi
```

Comment:
New P.sh projects may use another subdomain, i.e. eu-2.platform.sh

Reply:
I also think we don't need the conditional check for the presence on git remote named platform - it is internal to the platform CLI how to name it; we can just call this project:set-remote command and it will be fine - this is what I did in CWv2-ie